Makefile

```diff
@@ -456,3 +456,6 @@ generate_ci_workflows:
 
 release-notes:
 	go run ./go/tools/release-notes -from "$(FROM)" -to "$(TO)" -version "$(VERSION)" -summary "$(SUMMARY)"
+
+install_kubectl_kind:
+	./tools/get_kubectl_kind.sh
```

Comment:
I don't see this being used anywhere in the codebase

Reply:
This was used for the CI workflow which I have removed now. I have elected to keep all the other changes since it helps the next time we add the CI which should be soon enough